### DISCUSSION TITLE
Suggestion, web3/smart contract discord or telegram group to join

### DISCUSSION THREAD

**Discussion by king-null** [Cyfrin/foundry-full-course-cu#1552] [Ideas]
Do anyone have  a group where latest news or hack about web3 is been discuss also latest development 
and also a group where one can join and practice about smart contract

**Comment by EngrPips** [comment]
For latest hacks in the web3 space `rekt.news` is one of my favorite. for latest development in web3 `https://weekinethereumnews.com/` is my favorite and i think you can join cyfrin discord `https://discord.gg/EzA3CuF8` to discuss smart contracts

**Reply by raduom** [reply]
I think the invite has expired. Could we possibly get a new one?

**Reply by EngrPips** [reply]
Yeah sure `https://discord.gg/cyfrin`

**Reply by raduom** [reply]
That invite is invalid.

**Reply by raduom** [reply]
Nevermind. I think there may have been an issue with my browser.

**Reply by EngrPips** [reply]
Oh ok

**Comment by king-null** [comment]
thanks so much @EngrPips  please check the first link is not workinf please check if you probably made a mistake

**Reply by EngrPips** [reply]
Sorry for that, this is the correct link `https://rekt.news/`

**Comment by king-null** [comment]
hello @EngrPips  the link is still saying cant be reached, probable an error from the domain

**Reply by EngrPips** [reply]
I am not sure why the link is not working for you but i mean you can just browse `rekt.news` on google and it should suggest their website.
